samples/oauth_rails/VetsApiAppeals/app/views/nods/new.html.erb

```diff
@@ -0,0 +1,93 @@
+<h3 class='h3 text-center'>Submit <%= @schema['description'].gsub('Schema', '') %>
+  for <%= @veteran.present? ? "#{@veteran.name} requested by #{@name}" : @name %></h3>
+
+<div class="site-l-wrapper">
+  <div id='root'/>
+</div>
+
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+<script crossorigin src="https://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.21.1/babel.min.js"></script>
+<script crossorigin src="https://unpkg.com/react-jsonschema-form/dist/react-jsonschema-form.js"></script>
+
+<script type="text/babel">
+  let headerSchema = <%= raw @header_schema %>;
+  let schema = <%=raw @schema %>;
+
+  // Fix react-jsonschema-form not interpreting some header defs correctly
+  for (const d in headerSchema['definitions']) {
+    if ( /^X-/.test(d) ) {
+      headerSchema['definitions'][d]['type'] = 'string';
```

Comment:
Is this necessary since you've included `type: string` in the schema?

Reply:
I'm not sure! This was added by someone a whole lot more knowledgeable than I about the header-parser library used here ;) I can see what happens when I remove it.

Reply:
Looks like the rendering was fine either way.